service/src/test/java/greencity/service/ubs/UBSClientServiceImplTest.java

```diff
@@ -214,31 +212,21 @@ void getFirstPageData() {
             .price(170)
             .nameEng("NameEng")
             .locationId(1L)
-            .build();
-        BagTranslation bagTranslation = BagTranslation.builder()
-            .id(1L)
-            .name("Name")
-            .description("Description")
-            .bag(getBag().get())
-            .nameEng("NameEng")
-            .descriptionEng("DescriptionEng")
+            .limitedIncluded(false)
             .build();
 
         UserPointsAndAllBagsDto userPointsAndAllBagsDtoExpected =
-            new UserPointsAndAllBagsDto(List.of(bagTranslationDto), 600);
+            new UserPointsAndAllBagsDto(Collections.emptyList(), 600);
```

Comment:
you have changed the test logic
Previously we used `List.of(bagTranslationDto)` and now you are using `Collections.emptyList()`. Why?